src/test/java/integration/ShipmentGroupControllerIT.java

```diff
@@ -147,15 +149,15 @@ public void createShipmentGroup() throws Exception {
                         path("uuid");
         
         UUID newShipmentGroupId = UUID.fromString(newShipmentGroupIdString);
-        long timeFinished = currentTimeMillis();
+        LocalDateTime timeFinished = now();
 
         // check created data
         ShipmentGroup createdShipmentGroup = shipmentGroupService.getEntityById(newShipmentGroupId, user);
-        long timeCreated = createdShipmentGroup.getCreated().getTime();
-        long timeModified = createdShipmentGroup.getLastModified().getTime();
+        LocalDateTime timeCreated = createdShipmentGroup.getCreated();
+        LocalDateTime timeModified = createdShipmentGroup.getLastModified();
 
-        assertTrue(WRONG_CREATED_MESSAGE, timeFinished >= timeCreated && timeCreated >= timeStarted);
-        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished >= timeModified && timeModified >= timeStarted);
+        assertTrue(WRONG_CREATED_MESSAGE, timeFinished.isAfter(timeCreated) && timeCreated.isAfter(timeStarted));
+        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished.isAfter(timeModified) && timeModified.isAfter(timeStarted));
```

Comment:
Plz, change. isAfter is not enough. It should do ">=".
Plz, create helper method assertDateBetween(date, dateFrom, dateTo) in the new class AssertHelper(package util).